LABS.md

```diff
@@ -1,6 +1,6 @@
 ## 0. install git
 
-download: git-scm.com, or use your favourite package manager
+download: git-scm.com (https://git-scm.com), or use your favourite package manager
```

Comment:
the url should just replace the first part or you do it the markdown way `[git-scm.com](https://...)`